src/Italy/Mews.Fiscalizations.Italy/Dto/DtoUtils.cs

```diff
@@ -26,33 +23,20 @@ public static string NormalizeZip(this string zip)
             return Regex.Replace(zip, "[^0-9]", "");
         }
 
-        public static string NormalizeString(this string s, bool extendedAscii = true)
+        public static string NonEmptyValueOrNull(this string s)
         {
-            if (String.IsNullOrWhiteSpace(s))
-            {
-                return s.NonEmptyValueOrNull();
-            }
-
-            var highestCharacter = extendedAscii ? 255 : 127;
-            var strippedValue = StripDiacritics(s);
-            var chars = s.Select((c, i) => GetCharacterSubstitute(c, GetCharacterSubstitute(strippedValue[i], UnsupportedCharacterSubstitute, highestCharacter), highestCharacter)).ToArray();
-            return new String(chars);
+            return String.IsNullOrEmpty(s) ? null : s;
         }
 
-        public static string NonEmptyValueOrNull(this string str)
+        public static string StripDiacritics(this string s)
         {
-            return String.IsNullOrEmpty(str) ? null : str;
-        }
-
-        private static char GetCharacterSubstitute(char character, char substituteCharacter, int highestValidCharacter)
-        {
-            return (int)character <= highestValidCharacter ? character : substituteCharacter;
-        }
+            if (string.IsNullOrWhiteSpace(s))
+            {
+                return s.NonEmptyValueOrNull();
```

Comment:
Why would we replace empty string with a null?

Reply:
Yea that looks fishy, i just left as is for now, but will have a ticket to check the tech documentation and see if its needed or not..